Makefile

```diff
@@ -9,20 +9,29 @@ help: Makefile
 
 ## test:        run automated test suite
 test:
-	pytest --cov=yeat -m 'not long and not bandage'
+	pytest --cov=yeat -m 'not long and not bandage and not hifi'
 
 ## testlong:    run only long-running automated tests
 testlong:
-	pytest --cov=yeat -m 'long and not bandage'
+	pytest --cov=yeat -m 'long and not bandage and not hifi'
 
 ## testbandage: run only bandage required automated tests
 testbandage:
 	pytest --cov=yeat -m bandage
 
+## testhifi:    run only PacBio HiFi-reads automated tests
+testhifi:
+	pytest --cov=yeat -m 'hifi and not long and not bandage'
+
```

Comment:
New make command: `testhifi`. This command will only run assemblers that use the pacbio hifi-reads. These tests are extremely long and can take up to an hour depending on the computers resources.